docs/README.md

```diff
@@ -1,10 +1,59 @@
 [Return to Project Root](../)
 
 # User Documentation
-tbd
+sail-operator manages the lifecycle of your Istio control planes and aims to become a vehicle for automated integration of Istio with compatible third-party software. Instead of creating a new configuration schema, sail-operator APIs are built around Istio's helm chart APIs. All installation and configuration options that are exposed by Istio's helm charts are available through the sail-operator CRDs' `values` fields.
 
 ## Concepts
-tbd
+
+### Istio resource
+The `Istio` resource is used to manage your Istio control planes. You can access all helm chart options through the `values` field in the `spec`:
```

Comment:
Maybe adding a note that it's a cluster-wide resource?

Reply:
done! I also added a note to the IstioCNI section